packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/functions/AnyFunction.kt

```diff
@@ -16,7 +16,7 @@ abstract class AnyFunction(
   private val desiredArgsTypes: Array<AnyType>,
   isSync: Boolean = false
 ) {
-  internal var isSync = isSync
+  var isSync = isSync
```

Comment:
I don't like this change. I'm going to split the implementation of sync and async functions anyway. So for now it can stay ;)

Reply:
I don't like it too, but I haven't found any other simple way of doing it so far.
But you're going to refractor it anyway, it is a good deal for now